src/modules/Likes.js

```diff
@@ -0,0 +1,52 @@
+export default class Likes {
+  constructor(itemId = '', url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/O7NaeKDfq7oZ1XEIy2sV/likes/') {
+    this.itemId = itemId;
+    this.url = url;
+    this.liked = localStorage.getItem('liked') != null
+      ? JSON.parse(localStorage.getItem('liked'))
+      : '';
+    this.data = {
+      itemId: this.itemId,
+    };
+  }
+
+  saveLiked(TSave = this.liked) {
+    const storeLiked = JSON.stringify(TSave);
+    localStorage.setItem('liked', storeLiked);
+  }
+
+  async getLikes() {
+    const response = await fetch(this.url);
+    const json = await response.json();
+    return json;
+  }
+
+  async postLikes() {
+    const option = {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+      body: JSON.stringify(this.data),
+    };
```

Comment:
At the moment, a new like for the recipes is reflected only after clicked/liked, but on a refresh, it won't be reflected anymore. My like is lost :(

- [ ] Kindly check if the problem is that the like is not POSTed correctly to the Involvement API or that is not GET correctly. Narrow down the error by excluding possibilities of error one by one.